app/src/main/java/com/hieuminh/chatapp/constant/Constant.kt

```diff
@@ -0,0 +1,8 @@
+package com.hieuminh.chatapp.constant
+
+class Constant {
```

Comment:
change to object
example: object Constant {
object PrefKey {
const val USER_ID = "USER_ID"
}
}